module_utils/xpath_test.py

```diff
@@ -144,7 +163,7 @@ def test1d(self):
         nodes = linker._resolve('/port[name="port 1"]')
         assert nodes.count() == 1 and nodes.get(0) == self.port1
 
-    def test1d(self):
+    def test1e(self):
```

Comment:
why?

Reply:
there are 2 functions with same name test1d(self), so rename one test1d() to test1e()